src/VisualStudio/Core/Def/Implementation/Remote/ServiceHubRemoteHostClient.Connections.cs

```diff
@@ -17,17 +17,17 @@ internal sealed partial class ServiceHubRemoteHostClient : RemoteHostClient
     {
         internal static class Connections
         {
-            /// <summary>
-            /// call <paramref name="funcAsync"/> and retry up to <paramref name="timeout"/> if the call throws
-            /// <typeparamref name="TException"/>. any other exception from the call won't be handled here.
-            /// </summary>
-            public static async Task<TResult> RetryRemoteCallAsync<TException, TResult>(
+            private static readonly TimeSpan s_reportTimeout = TimeSpan.FromMinutes(10);
+            private static bool s_timeoutReported = false;
+
+            private static async Task<Stream> RequestServiceWithCancellationRetryAsync(
                 Workspace workspace,
-                Func<Task<TResult>> funcAsync,
+                HubClient client,
+                ServiceDescriptor descriptor,
                 TimeSpan timeout,
-                CancellationToken cancellationToken) where TException : Exception
+                CancellationToken cancellationToken)
             {
-                const int retry_delayInMS = 50;
+                const int RetryDelayInMS = 50;
```

Comment:
💡 If you are already changing these lines, it would be good to make them `TimeSpan` at the same time.